src/zcash/IncrementalMerkleTree.hpp

```diff
@@ -121,13 +123,22 @@ class SHA256Compress : public uint256 {
     static SHA256Compress combine(const SHA256Compress& a, const SHA256Compress& b);
 };
 
+/* The old performance optimization, compatible with the libzerocash tree. */
```

Comment:
I want to point out that the fixpoint was not a performance optimization in any way, despite my comment which suggested otherwise.